zstash/extract.py

```diff
@@ -182,18 +182,23 @@ def extract(keep_files=True):
             cur.execute(u"select value from config where arg=?", (attr,))
             value = cur.fetchone()[0]
             setattr(config, attr, value)
-    # FIXME: No overload variant of "int" matches argument type "None" mypy(error)
-    config.maxsize = int(config.maxsize)  # type: ignore
-    # FIXME: Incompatible types in assignment (expression has type "bool", variable has type "None")mypy(error)
-    config.keep = bool(int(config.keep))  # type: ignore
+    if config.maxsize:
+        maxsize = config.maxsize
+    else:
+        raise Exception("Invalid config.maxsize={}".format(config.maxsize))
+    config.maxsize = int(maxsize)
+    if config.keep:
+        keep = config.keep
+    else:
+        raise Exception("Invalid config.keep={}".format(config.keep))
```

Comment:
You should use raise ValueError(...) to be more explicit on what the exception is.

https://docs.python.org/3/library/exceptions.html#ValueError
https://stackoverflow.com/a/43605619